packages/@okta/vuepress-site/docs/release-notes/2022-okta-identity-engine/index.md

```diff
@@ -6,6 +6,20 @@ title: Okta Identity Engine API Products release notes 2022
 
 ## January
 
+### Weekly release 2022.01.1
+
+| Change                                                                                              | Expected in Preview Orgs |
+|-----------------------------------------------------------------------------------------------------|--------------------------|
+| [Bugs fixed in 2022.01.1](#bugs-fixed-in-2022-01-1)         | January 13, 2022           |
+
+#### Bugs fixed in 2022.01.1
+
+* If the [Create a Scope](/docs/reference/api/authorization-servers/#create-a-scope) endpoint received multiple requests at or near the same time, duplicate Scopes could be created. (OKTA-442533)
```

Comment:
duplicate Scopes << should "Scopes" be initial uppercase here? I've noticed that scopes is usually lowercase in the dev docs, or is this special?

Reply:
Good question. It's capitalized in the api docs to which I link. That's what prompted me to capitalize it here. I see that we don't capitalize it in the guides. I think I'll capitalize the link text to align with the reference doc but leave the other use of scope in lowercase.